lib/pymedphys/_gamma/implementation/shell.py

```diff
@@ -309,6 +316,15 @@ def from_user_inputs(
 
 
 def gamma_loop(options: GammaInternalFixedOptions):
+
+    # TODO: warning here too or no?
```

Comment:
Ahh, I see what you mean (ignore my other comment).

We don't expose gamma loop directly via the documented pymedphys API, so, I think we can skip needing that notice here.

Reply:
So just in `gamma_shell()` then?

Reply:
Yup, that's the only function here that we have exposed as a part of the PyMedPhys public API:

https://github.com/pymedphys/pymedphys/blob/ea9bfe6d6555a2a6d73da4fa6c0290dc0f644f98/lib/pymedphys/__init__.py#L6

Reply:
In the future, if there is an issue of wanting to avoid multiple warning, another approach would have been to put the warning inside a separate function, and then just call that function everywhere where relevant. I believe that would make it so that it only ends up emitting once.

But, I don't think that approach is needed here.

Reply:
Ah yes, that makes sense.